praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def reply(self, body, author_hidden=False, internal=False):
+        """Reply to the conversation.
+
+        :param body: The markdown formatted content for a message.
+        :param author_hidden: When True, author is hidden from non-moderators.
+        :param internal: When True, message is hidden from non-moderators.
```

Comment:
Looks like the authors on "Internal" messages can be marked hidden or not-hidden. This does have an effect, sort-of: the little badge showing that the author is hidden will show up, even though everyone reading an internal message is a moderator and can therefore see who the author is.